### ISSUE TITLE
Segfaults since commit 1da7077b3da0c6f139e6b0999c3c24ebb144b9c1

### ISSUE THREAD

**Issue by mattpepin** [moddevices/mod-host#66]
Since commit 1da7077b3da0c6f139e6b0999c3c24ebb144b9c1, I cannot do:

add http://calf.sourceforge.net/plugins/Equalizer5Band 0

mod-host segfaults

**Comment by falkTX** [comment]
can you debug the issue please?

**Comment by mattpepin** [comment]
Yes probably. I reverted to this commit for now, but I will try to open a merge request later.
